docs/modules/node.md

```diff
@@ -191,31 +190,6 @@ none
 #### Returns
 system heap size left in bytes (number)
 
-## node.info()
-
-Returns NodeMCU version, chipid, flashid, flash size, flash mode, flash speed.
-
-#### Syntax
-`node.info()`
-
-#### Parameters
-none
-
-#### Returns
- - `majorVer` (number)
- - `minorVer` (number)
- - `devVer` (number)
- - `chipid` (number)
- - `flashid` (number)
- - `flashsize` (number)
- - `flashmode` (number)
- - `flashspeed` (number)
-
-#### Example
-```lua
-majorVer, minorVer, devVer, chipid, flashid, flashsize, flashmode, flashspeed = node.info()
-print("NodeMCU "..majorVer.."."..minorVer.."."..devVer)
-```
```

Comment:
This actually fixes #2759 in a way 😄 However, I'd rather keep the issue open as `node.info()` would still be useful to have I guess.

Reply:
I wasn't even consciously aware of that issue. The fact that there is documentation for a function that has never been implemented on ESP32 is a result of copy/paste failure (likely mine) from the esp8266 branch. I agree that leaving the issue open is fair, but I would suggest that removing the documentation is the right thing to until such a time someone does implement such a function.

Reply:
I fully agree.